src/services/completions.ts

```diff
@@ -675,6 +675,37 @@ namespace ts.Completions {
             hasAction = !importCompletionNode;
         }
 
+        const kind = SymbolDisplay.getSymbolKind(typeChecker, symbol, location); // TODO: GH#18217

```

Comment:
There's no type assertion here, right?

```suggestion
        const kind = SymbolDisplay.getSymbolKind(typeChecker, symbol, location);
```

Reply:
Done.